Makefile

```diff
@@ -13,31 +13,69 @@
 # limitations under the License.
 
 GO := go
-
 EPOCH_TEST_COMMIT := f98810348b4b26d8c4dd64e36bde613049b2b0b9
+PROJECT := github.com/IBM/cap
+BUILD_DIR := _output
+SOURCES := $(shell find go/atom go/cap go/captn go/shared vendor/ -name '*.go')
 
 default: help
 
 help:
 	@echo "Usage: make <target>"
 	@echo
-	@echo " * 'verify'           	- Execute the source code verification tools"
-	@echo " * 'install.tools'    	- Install tools used by verify"
+	@echo " * 'binaries'      - Build binaries"
+	@echo " * 'clean'         - Clean build artifacts"
+	@echo " * 'test'          - Run Unit and Integration Tests"
+	@echo " * 'unit'          - Run Unit Tests"
+	@echo " * 'coverage'      - Determine Unit and Integration Test Coverage, generates profile in coverage.out"
+	@echo " * 'verify'        - Execute the source code verification tools"
+	@echo " * 'install.tools' - Install tools used by verify"
+
+.PHONY: binaries clean test unit integration coverage
+
+binaries: $(BUILD_DIR)/captn
+
+$(BUILD_DIR)/captn: $(SOURCES)
+	$(GO) build -o $@ \
+		-gcflags '$(GO_GCFLAGS)' \
+		$(PROJECT)/go/captn
+
+clean:
+	rm -rf $(BUILD_DIR)/*
 
-.PHONY: verify
+test: ## test the go packages unit and integration
+	$(GO) test ./go/atom ./go/cap ./go/shared -v -tags=integration
 
-verify: .gitvalidation
+unit: ## test the go packages
+		$(GO) test ./go/atom ./go/cap ./go/shared -v
+
+coverage: ## test and determine coverage of the go packages
+	$(GO) test ./go/atom ./go/cap ./go/shared -tags=integration -covermode=count -coverprofile=$(BUILD_DIR)/coverage.out
+
+.PHONY: verify gofmt golint
+
+verify: .gitvalidation gofmt golint
 
 .PHONY: .gitvalidation
```

Comment:
this thingy ain't working for me. I'll troubleshoot.

Reply:
Did you run the make install.tools?

Reply:
I'm gonna go with transient weirdness on a `go get` or some such.